### PR TITLE
Fix mysql

### PR DESCRIPTION
**Pull request by M0ses**

### CODE REVIEW

**Comment by ChrisBr** [comment]
@M0ses can you give some explanation why this is broken?
Is this also responsible for the broken packages in O:S:U?

**Comment by M0ses** [comment]
The idea is to have all relevant OBS relevant data under the directory /srv/obs. So if you use the appliance with an additional disk image containing a lvm vg called "OBS", you should be able to "throw away" the appliance, use a newer appliance and attach you data disk, boot up and you are done without updating each packet.

**Comment by M0ses** [comment]
No, It is not a fix for broken package.

**Comment by ChrisBr** [comment]
Ok, thanks :+1:

File: src/api/config/database.yml.example [hunk 1]
```diff
@@ -14,6 +14,7 @@ production:
   encoding: utf8
   timeout: 15
   pool: 30
+  socket: /var/run/mysql/mysql.sock
```
Comment (hennevogel) [inline src/api/config/database.yml.example]: That is the default. Why do you have to explicitly set this?

**Comment by M0ses** [comment]
Will run test installations for sles12/leap again

**Pull request merged**